media/css/firefox/personal.less

```diff
@@ -0,0 +1,194 @@
+@import "../sandstone/lib.less";
+
+@font-face {
+    font-family: 'FiraSans Regular';
+    src: url('/media/fonts/FiraSans-Regular.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'FiraSans Bold';
+    src: url('/media/fonts/FiraSans-Bold.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'FiraSans Light';
+    src: url('/media/fonts/FiraSans-Light.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+body {
+    font-family: 'FiraSans Regular'
+}
+
+#outer-wrapper {
+    background: none;
+}
+#wrapper {
+    width: 100%;
+}
+#masthead {
+    background: #0c99d5 url(/media/img/firefox/personal/header.png) 0 0 no-repeat;
+
+    @media only screen and (min-width: @breakTablet) and (max-width: @breakDesktop) {
+        width: 100%;
+    }
+}
+.header-content {
+    margin: 0 auto;
+    padding: 40px 0 70px 0;
+    width: 100%;
+    max-width: 1400px;
+    text-align: center;
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        padding-bottom: 60px;
+    }
+
+    h1 {
+        color: #bee1f5;
+        margin: 0 0 55px 0;
+        font-family: 'FiraSans Bold';
+        .font-size(5rem);
+        font-weight: bold;
+        line-height: 1.1;
+        text-transform: uppercase;
+        letter-spacing: 1px;
+        text-shadow: none;
+
+        @media only screen and (max-width: @breakDesktop) {
+            .font-size(3rem);
+        }
+
+        @media only screen and (max-width: @breakTablet) {
+            font-family: 'FiraSans Regular';
+            .font-size(2.5rem);
+            letter-spacing: -1px;
+        }
+
+        span {
+            display: block;
+            color: #fff;
+            margin-left: 28%;
+
+            @media only screen and (max-width: @breakMobileLandscape) {
+                margin-left: 0;
+            }
+        }
+    }
+    h1:before {
+        display: block;
+        background: transparent url(/media/img/firefox/personal/logo.png) 0 0 no-repeat;
+        margin: 0 auto @baseLine;
+        width: 147px;
+        height: 147px;
+        content: '';
+    }
+    .download-button.download-button-simple {
+
+        @media only screen and (max-width: @breakMobileLandscape) {
+            display: none;
+        }
+
+        .download-link {
+            width: 260px;
+        }
+    }
+    .download-content {
+        text-transform: uppercase;
+    }
+}
+main {
+    margin: 0 auto;
+    padding: 60px 0 150px 0;
+    width: 100%;
+    max-width: 1400px;
+    text-align: center;
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        padding-bottom: 60px;
+    }
+}
+.mission {
+    display: inline-block;
+    margin-left: 50px;
```

Comment:
Instead of adding a left margin value to horizontal lists like these, could you just add some padding to either side maybe? (e.g. `padding: 0 25px;`). Then you wouldn't need to remove the left margin using `first-child` which isn't supported in older browsers.